### PR TITLE
Fix errors in the Python tutorial

### PR DESCRIPTION
**Pull request by mprahl**
The line numbers were incorrect. Most were off by 1. This is partially due to 85347f3dc8a7c598e7203aeac61ffe72325a0809.

This commit also adds some missing commas and fixes some typos.

### CODE REVIEW

File: python/docs/tutorial.rst [hunk 1]
```diff
@@ -249,17 +248,17 @@ Now let's create a simple client to test this service out.
    :linenos:
 
 As well as sending requests, we need to be able to get back the
-responses. We create a receiver for that (see line 14), but we don't
+responses. We create a receiver for that (see line 15), but we don't
 specify an address, we set the dynamic option which tells the broker
-we are connected to to create a temporary address over which we can
+we are connected to a temporary address over which we can
```
Comment (ssorj) [inline python/docs/tutorial.rst]: For this particular sentence, the original is correct, though a little unclear.  Simplified, the sense is: the option tells the broker to create an address.

**Comment by mprahl** [comment]
Thanks @ssorj. How does this look?

**Comment by ssorj** [comment]
That looks great!  Ship it, I say.

**Comment by gemmellr** [comment]
I raised https://issues.apache.org/jira/browse/PROTON-2165 and committed these changes against it.

The changes will only be part of the next release after 0.30.0,  but I have included them in the website updates for 0.30.0 via https://github.com/apache/qpid-site/commit/513a29d5b516df102a994b60f256ed57914b341a.

**Pull request closed without merging**